### DISCUSSION TITLE
about the affects c++ interface of tensorflow may make

### DISCUSSION THREAD

**Discussion by dxy-cs** [deepmodeling/deepmd-kit#1473] [Q&A] [answered]
We have installed the python interface of tensorflow, and deepmd-kit can run successfully. However, we didn't installed the c++ interface of tensorflow, because there is much difficulties when compile it. So we want to ask, will it affect the performance of deepmd-kit's training process if c++ interface is missing？

**Comment by wanghan-iapcm** [comment, marked as answer]
The answer is no. The training only uses the python interface.
